tests/aws/services/kms/test_kms.py

```diff
@@ -540,7 +540,7 @@ def test_sign_verify(self, kms_create_key, snapshot, key_spec, sign_algo, aws_cl
             )
         assert exc.match("ValidationException")
 
-    @markers.aws.unknown
+    @markers.aws.validated
```

Comment:
Strange, the tests already had a validated timestamp, but were not flagged as validated.

Easy wins, then! :smile:

Reply:
Yeah, It was strange to me as well in the first place and so I ran them twice to confirm if I am missing on something 😄